modules/sw360/sw360-client/src/main/java/org/eclipse/sw360/antenna/sw360/client/rest/resource/releases/SW360Release.java

```diff
@@ -386,4 +386,34 @@ public int hashCode() {
     public boolean canEqual(Object o) {
         return o instanceof SW360Release;
     }
+
+    public String csvPrintHeader() {
+        return "release id;name;version;" +
+                "coordinates;" +
+                "main license ids;" +
+                "overridden license;declared license;observed license;" +
+                "copyrights;clearing state;change status;" +
+                "download url; release tag url; software heritage id";
+    }
+
+    public String csvPrintRow() {
+        return String.format("%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;%s;%s",
+                getId(), name, version,
+                collectionToString(getCoordinates().values()),
+                collectionToString(getMainLicenseIds()),
+                getOverriddenLicense(), getDeclaredLicense(), getObservedLicense(),
+                getCopyrights(), getClearingState(), getChangeStatus(),
+                getDownloadurl(), getReleaseTagUrl(), getSoftwareHeritageId());
+    }
+
+    private String collectionToString(Collection<String> collection) {
+        String separator = "";
+        StringBuilder mainLicenses = new StringBuilder();
+        for (String item : collection) {
+            mainLicenses.append(separator);
+            separator = ",";
+            mainLicenses.append(item);
+        }
+        return mainLicenses.toString();
+    }
```

Comment:
I am not sure whether these format-specific output methods should be part of the data model. I would rather keep this separate, e.g. as part of the ReportOutputCSV implementation. When we support other output formats, would we add corresponding methods as well?

Reply:
oh I agree. I should have mentioned that I regarded this as an intermediate solution until i come up with a better idea for the demo.